### PR TITLE
Feature/new game overpage

### LINKED ISSUE #78: New Game Over Screen
**As a** user **I want a** game over screen that follows the app desing **since** this looks better and I like nice things ;) 

# Tasks
* [x] Lists all tasks (with id ex. #16) from potential break down

# Acceptance Criteria
* [ ] Create a new Game Over screen with the same desing patterns as the rest of the app
* [ ] The User should be able to start a new game and view how many point he/she got

# Definition of done
* [ ] Code compiles
* [ ] Passes all tests
* [ ] Reviewer runs code on own machine
  * [ ] Reviewer 1
  * [ ] Reviewer 2

### PR DESCRIPTION
**Pull request by malinrudal**
Fixes #78

### CODE REVIEW

**Comment by DanielWillim** [comment]
To fix bug #100 :
- Rewrote Question to be a class and use componentDidMount & willUnmount
- Changed setTimeOut -> setIntervall
- Timers value is stored in app but updated through tick function in Question
- timer is now stored in seconds not miliseconds removing need to round number before display

File: src/Answer.jsx [hunk 1]
```diff
@@ -29,56 +32,54 @@ function Answer({
   answers,
```
Comment (malinrudal) [inline src/Answer.jsx]:
```suggestion
  answers,
  buttontext,
```

File: src/Answer.jsx [hunk 2]
```diff
@@ -29,56 +32,54 @@ function Answer({
...
+                {`Tid kvar: ${timer === undefined ? 0 : timer}`}
+              </Typography>
+            </Grid>
+            <Grid item xs={6}>
+              <Typography variant="body1" align="right">
+                {`Poäng: ${points}`}
+              </Typography>
+            </Grid>
+          </Grid>
+          <Button variant="contained" align="center" onClick={onNext}>Nästa fråga</Button>
```
Comment (malinrudal) [inline src/Answer.jsx]:
```suggestion
          <Button variant="contained" align="center" onClick={onNext}>{buttontext}</Button>
```

File: src/App.jsx [hunk 3]
```diff
@@ -148,17 +146,21 @@ class App extends Component {
 
     if (gameOver) {
       return (
-        <GameOver
+        <Answer
           onNext={this.restartQuestions}
+          answers={answers}
           answer={correctAnswers}
```
Comment (malinrudal) [inline src/App.jsx]:
```suggestion
          answer={correctAnswers}
          buttontext="Starta nytt spel"
```

File: src/App.jsx [hunk 4]
```diff
@@ -148,17 +146,21 @@ class App extends Component {
...
+          category="Lokalområde"
```
Comment (malinrudal) [inline src/App.jsx]:
```suggestion
          category="Tiden är slut"
          
```

File: src/App.jsx [hunk 5]
```diff
@@ -148,17 +146,21 @@ class App extends Component {
...
+          question={question}
```
Comment (malinrudal) [inline src/App.jsx]:
```suggestion
          question="Game Over"
```

File: src/App.jsx [hunk 6]
```diff
@@ -148,17 +146,21 @@ class App extends Component {
...
       );
     }
 
+    // Fail
     return (
       <Answer
-        mening="Fail! Du svarade fel!"
         onNext={this.nextQuestion}
         answers={answers}
         answer={correctAnswers}
```
Comment (malinrudal) [inline src/App.jsx]:
```suggestion
        answer={correctAnswers}
        buttontext="Nästa fråga"
```

File: src/App.jsx [hunk 7]
```diff
@@ -133,7 +132,6 @@ class App extends Component {
     if (responded.won) {
       return (
         <Answer
-          mening="Grattis du svarade rätt!"
           onNext={this.nextQuestion}
           answers={answers}
           answer={correctAnswers}
```
Comment (malinrudal) [inline src/App.jsx]:
```suggestion
          answer={correctAnswers}
          buttontext="Nästa fråga"

```

File: src/Question.jsx [hunk 8]
```diff
@@ -1,78 +1,103 @@
...
-  onAnswer,
-  onTimeOut,
-  question,
-  timer,
-  viewTimeLeft,
-}) {
-  const timeWhenStarted = (new Date()).getTime();
-  const timerFloor = timer / 1000;
+// How many seconds each tick should be
+const GAME_TIMER_INTERVALL = 0.1;
```
Comment (Kirens) [inline src/Question.jsx]: See comment on L87
```suggestion
const GAME_TIMER_PRECISION = 1;

```

**Pull request merged**